libraries/AP_HAL_ChibiOS/hwdef/ORBITH743/README.md

```diff
@@ -0,0 +1,202 @@
+
+## DShot Capability
+
+All motor outputs (M1–M8) support:
+- DShot
+- Bi-directional DShot (for BIDIR motors)
+- PWM
+
+BIDIR DShot is supported on:
+- M1 (PA0)
+- M3 (PA2)
+- M5 (PB0)
+- M7 (PB4)
+
+> **Important:** Mixing DShot and PWM within the same timer group is **not allowed**. Groups must be uniformly configured. Timer groups are:  
+1/2, 3/4, 5/6, 7/8.
+
+Servo outputs (S1–S4) on PA15, PB3, PD12, and PD13 (TIM2 and TIM4 timers) are PWM only. Output 13 (LED) is in a separate group and not DShot-capable.
+
+## GPIOs
+
+ORBITH743 outputs can be used as GPIOs (relays, buttons, RPM, etc.).  
+Set the `SERVOx_FUNCTION = -1` to enable GPIO functionality. See [ArduPilot GPIO docs](https://ardupilot.org) for more info.
+
+### GPIO Pin Mapping
+
+- PWM1 → 50  
+- PWM2 → 51  
+- PWM3 → 52  
+- PWM4 → 53  
+- PWM5 → 54  
+- PWM6 → 55  
+- PWM7 → 56  
+- PWM8 → 57  
+- PWM9 → 58  
+- PWM10 → 59  
+- PWM11 → 60  
+- PWM12 → 61  
+- LED → 62  
+- BUZZER → 80  
+- PINIO1 → 81 (internal)  
+- PINIO2 → 82 (internal)
+
+## VTX Power Control
+
+GPIO 81 controls the 10V VTX BEC output.  
+Setting GPIO 81 **low** disables voltage to the pins.
+
+Example (using Channel 10 to toggle VTX BEC using Relay 2):
+```text
+RELAY2_PIN = 81  
+RC10_OPTION = 34  ; Relay2 Control
+```
+
+> ⚠️ **Warning:** PINIO1 controls the 10V DC-DC converter (HIGH = on, LOW = off). Default: ON. Always install an antenna on the VTX when battery-powered.
+
+## Camera Switch Control
+
+GPIO 82 controls camera switching via PINIO2. Set high or low to toggle between cameras (e.g., front/rear).
+
+Example (using Channel 11 to control camera switch via Relay 2):
+
+```text
+RELAY2_PIN = 82  
+RC11_OPTION = 34  ; Relay2 Control
+```
+
+> ⚠️ **Warning:** PINIO2 toggles camera input (HIGH/LOW). Ensure wiring matches desired switching behavior.
+
+## Connecting a GPS/Compass Module
+
+This board does **not** include GPS or compass modules. An [external GPS/compass](https://ardupilot.org) must be connected for autonomous features.
+
+> **Note:** If GPS is powered via 5V and connected to UART6, a battery is required for power.
+
+> **Tip:** The 4V5 pin can power both RC and GPS for bench setup (without battery), as long as the total current does not exceed USB limits (typically 1A).
+
+## Battery Monitor Settings
+
+These are set by default. If reset:
+
+Enable battery monitor with:
+
+```text
+BATT_MONITOR = 4
+```
+
+Then reboot.
+
+**First battery monitor:**
+
+* `BATT_VOLT_PIN = 10`
+* `BATT_CURR_PIN = 11`
+* `BATT_VOLT_MULT = 10.1`
+* `BATT_AMP_PERVLT = 80.0` *(Calibrate as needed)*
+
+**Second battery monitor:**
+
+* `BATT2_VOLT_PIN = 4`
+* `BATT2_CURR_PIN = 18`
+* `BATT2_VOLT_MULT = 10.1`
+* `BATT2_AMP_PERVLT = 80.0` *(Calibrate as needed)*
+
+> **Note:** The autopilot uses a high-precision current sensor that is sensitive to ESC noise. Add low-ESR capacitors if needed.
```

Comment:
there is no current sensor in your autopilot....this is for escs or onboard current sensors using certain i to v chips...remove